lib/routes/mangadex/user/follows.ts

```diff
@@ -0,0 +1,156 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import getToken from '../_access';
+import cache from '@/utils/cache';
+import { config } from '@/config';
+import { getMangaChapters, getMangaMetaByIds } from '../_feed';
+
+type FollowType = 'reading' | 'plan-to-read' | 'completed' | 'on-hold' | 're-reading' | 'dropped';
+type StatusType = 'reading' | 'plan_to_read' | 'completed' | 'on_hold' | 're_reading' | 'dropped';
+type LabelType = 'Reading' | 'Plan to Read' | 'Completed' | 'On Hold' | 'Re-reading' | 'Dropped';
+
+const statusMap: Record<FollowType, StatusType> = {
+    reading: 'reading',
+    'plan-to-read': 'plan_to_read',
+    completed: 'completed',
+    'on-hold': 'on_hold',
+    're-reading': 're_reading',
+    dropped: 'dropped',
+};
+
+const labelMap: Record<FollowType, LabelType> = {
+    reading: 'Reading',
+    'plan-to-read': 'Plan to Read',
+    completed: 'Completed',
+    'on-hold': 'On Hold',
+    're-reading': 'Re-reading',
+    dropped: 'Dropped',
+};
+
+export const route: Route = {
+    path: '/user/follow/:type?',
+    name: "Logged User's Followed Mangas Feed",
+    maintainers: ['chrisis58'],
+    example: '/mangadex/user/follow/reading',
+    description: `Fetches the feed of mangas that you follow on MangaDex whick are in the specified status.
+CAUTION: With big amount of follows, it may take a long time to load or even fail.
+It's recommended to use the \`/mangadex/mdlist/:listId?\` route instead for better performance, though it requires manual configuration.`,
+    categories: ['anime'],
+    parameters: {
+        type: {
+            description: 'The type of follows to fetch',
+            default: 'reading',
+            options: [
+                { value: 'reading', label: 'Reading' },
+                { value: 'plan-to-read', label: 'Plan to Read' },
+                { value: 'completed', label: 'Completed' },
+                { value: 'on-hold', label: 'On Hold' },
+                { value: 're-reading', label: 'Re-reading' },
+                { value: 'dropped', label: 'Dropped' },
+            ],
+        },
+    },
+    radar: [
+        // TODO: how to resolve the query string?
+        {
+            source: ['mangadex.org/titles/follows', 'mangadex.org/titles/follows?tab=reading'],
```

Comment:
```suggestion
            source: ['mangadex.org/titles/follows'],
```
`source` can not be used to match search parameter